api/nodes/serializers.py

```diff
@@ -459,6 +473,15 @@ def get_registration_links_count(self, obj):
                 count += 1
         return count
 
+    def get_linked_by_nodes_count(self, obj):
+        return NodeRelation.objects.filter(child=obj, is_node_link=True, parent__is_deleted=False, parent__type='osf.node').count()
```

Comment:
Always use the `related_name` when possible, in order to take advantage of Django's query caching.

```python
    child = models.ForeignKey('AbstractNode', related_name='_parents', on_delete=models.CASCADE)


obj._parents.filter(....)
```